docs/reference/2-ods-api/how-to-guides/how-to-deploy-an-extension-plugin.md

```diff
@@ -110,29 +170,37 @@ You can also verify the Sample API endpoints in the Swagger UI documentation:
     ```powershell
     Import-Module .\Deployment.psm1
     Initialize-DeploymentEnvironment
```

Comment:
I don't know why this is all highlighted in red. Yuck.

Reply:
https://github.com/github-linguist/linguist/issues/6668

Seems like it's an odd but common occurrence

Reply:
Hmm Im not seeing this.